packages/server/lib/server.coffee

```diff
@@ -70,7 +70,14 @@ class Server
 
     ## set the cypress config from the cypress.json file
     app.set("view engine", "html")
-    app.engine("html",     hbs.__express)
+
+    ## since we use absolute paths, configure express-handlebars to not automatically find layouts
+    ## https://github.com/cypress-io/cypress/issues/2891
+    app.engine "html", exphbs {
```

Comment:
Just a stylistic thing, but we prefer to always use parentheses when calling functions in CoffeeScript. The only exception is tests, where we forgo them for `it`, `describe`, etc.

```coffeescript
app.engine("html", exphbs({
# ...
}))
```